packages/models/src/lib/global-cli-options.ts

```diff
@@ -0,0 +1,44 @@
+import { z } from 'zod';
+
+/**
+ * Define Zod schema for the GlobalCliArgs type
+ *
+ * @example
+ *
+ * // Example data for the GlobalCliArgs type
+ * const uploadConfigData = {
+ *   server: 'https://example.com/api',
+ *   apiKey: 'your-api-key',
+ * };
```

Comment:
This is an upload config example, but these docs should be for global CLI options.

Reply:
fixed